downstream/modules/navigator/proc-review-ee-cli.adoc

```diff
@@ -0,0 +1,301 @@
+
+[id="proc-review-ee-cli_{context}"]
+
+
+
+= Reviewing {ExecEnvNameStart} from the {Navigator} CLI
+
+[role="_abstract"]
+
+You can review your {ExecEnvNameStart} with the {Navigator} CLI. 
+
+.Prerequisites
+
+* {ExecEnvName}
+
+.Procedure
+
+* Review the {ExecEnvName} included in your {Navigator} configuration.
++
+```
+$ ansible-navigator ee-details -m stdout
```

Comment:
@samccann hello, @cidrblock are changing something on that. Probably we need to review after merge.